templates/datalab/infrastructure-authorisation.template.yml

```diff
@@ -5,17 +5,17 @@ metadata:
   namespace: {{ namespace }}
 ---
 apiVersion: rbac.authorization.k8s.io/v1beta1
-kind: Role
+kind: ClusterRole
 metadata:
   name: infrastructure-management-role
   namespace: {{ namespace }}
```

Comment:
As clusterroles are not namespaced, I suppose the namespace map key could be removed from the template

Reply:
Good point - have raised a new PR and fixed this there